### DISCUSSION TITLE
How to install on cPanel

### DISCUSSION THREAD

**Discussion by anthonycibin** [directus/directus#4998] [Archive]
This error occurs with cPanel

<img width="1736" alt="error" src="https://user-images.githubusercontent.com/81036840/114318067-8009bf00-9b0b-11eb-8a88-1568ede1028e.png">

**Comment by Jakob-em** [comment]
Which operating system are you using to run directus? 
Could be the same problem as described here: https://github.com/ranisalt/node-argon2/issues/276

**Comment by anthonycibin** [comment]
> Which operating system are you using to run directus?
> Could be the same problem as described here: [ranisalt/node-argon2#276](https://github.com/ranisalt/node-argon2/issues/276)

Thanks!

With this the first part started
`npm rebuild argon2 --build-from-source
`

Now on
`npx directus start`
this occurs

<img width="1675" alt="Schermata 2021-04-11 alle 21 54 57" src="https://user-images.githubusercontent.com/81036840/114319172-9108ff00-9b10-11eb-8b49-1235fbc2fab7.png">

**Reply by rijkvanzanten** [reply]
From a quick google it seems that the `CXXABI_1.3.8` error is fairly common and most likely caused by a missing package within your linux environment. It seems like `gcc` is the unix library that contains this, so you might have to update that 👍🏻
